site/content/docs/5.3/utilities/borders.md

```diff
@@ -122,6 +129,10 @@ Or, choose from any of the `.border-opacity` utilities:
 
 ## Width
 
+{{< design-callout-alert >}}
+The three last border utilities with **a width superior to 2px** should not be used because they do not respect the Orange Design System specifications.
```

Comment:
OK for the modification. I just would replace "a width superior to 2px" by one of these 3 propositions:
- "a width greater than 2px" 
- "a width larger than 2px"
- "a width of more than 2px"